examples/nextjs/grafbase/schema.graphql

```diff
@@ -0,0 +1,12 @@
+type TodoList @model {
+  id: ID!
+  title: String!
+  todos: [Todo]
+}
+
+type Todo @model {
+  id: ID!
+  title: String!
+  complete: Boolean
+  list: TodoList!
```

Comment:
remove bang here

Reply:
What do you mean? I remove this file?

Reply:
Remove the `!` to match the template

Reply:
Well in this case this is required, you can't create todos outside of a list
But I will update